assets/src/web-stories-block/css/views/carousel.css

```diff
@@ -19,24 +16,19 @@
   overflow: hidden;
 }
 
-.is-view-type-carousel .glider-next,
-.is-view-type-circles .glider-next {
+.is-carousel .glider-next {
```

Comment:
Just `.is-carousel` is very generic. What if a theme already uses `is-carousel` somewhere?

Shouldn't use ` .web-stories-list.is-carousel` or something?

Reply:
yep, I'll update this once.

Reply:
Updated, also added specificity for few other selectors like `.is-view-type-circles`.